### PR TITLE
feat: add python generator

### PR DESCRIPTION
**Pull request by jonaslagoni**
**Description**
This PR extends the already existing work from @mahakporwal02 in https://github.com/asyncapi/modelina/pull/604 and adds a python generator for both class and enums.

cc @liquidautumn @niraveen as you showed interest in the other PR, I am just pinging you in case you wish to review it 🙂 

**Related issue(s)**
Fixes https://github.com/asyncapi/modelina/issues/49

### CODE REVIEW

File: src/generators/python/renderers/ClassRenderer.ts [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+${renderer.indent(body)}`;
+  },
+  getter({ property }) {
+    return `@property
+def ${property.propertyName}(self):\n\treturn self._${property.propertyName}`;
+  },
+  setter({ property }) {
+    return `@${property.propertyName}.setter
+def ${property.propertyName}(self, ${property.propertyName}):\n\tself._${property.propertyName} = ${property.propertyName}`;
+  },
```
Comment (magicmatatjahu) [inline src/generators/python/renderers/ClassRenderer.ts:96]: Should we use decorators as default behaviour? I don't know Python at all, so I only ask.

**Review by magicmatatjahu — COMMENTED** [review]
One question related to code. It would be best if someone who knows Python does a review :)

File: src/generators/python/renderers/ClassRenderer.ts (hunk 1 above)
Comment (jonaslagoni) [reply src/generators/python/renderers/ClassRenderer.ts:96]: This was what https://github.com/asyncapi/modelina/pull/604 used so I just copied it 🙂 

You will also likely not even reach a consensus as it's preference

**Review by magicmatatjahu — APPROVED** [review]
Accept but I don't know Python so please wait for other reviewers :)

**Comment by jonaslagoni** [comment]
> Accept but I don't know Python so please wait for other reviewers :)

Why? 😄 There was no objections in the last pr and we have Black Box tests to ensure it works.

If something else is desired from the models we get those requests down the line. So i dont see any reason to wait 😄 Unless you know someone who knows python that wants to review it 🤔?

**Comment by jonaslagoni** [comment]
Gonna take the decision to just merge this now, as we can always backtrack in next generation if something is completely wrong.

**Pull request merged**